support-frontend/app/views/main.scala.html

```diff
@@ -58,6 +59,17 @@
       </style>
     }
 
+    @linkedJs(path: RefPath) = {
+      <script type="text/javascript" src="@assets(path)"></script>
+    }
+
+      @inlineJs(inlineJs: StyleContent) = {
+        <script type="text/javascript">
```

Comment:
one thing I'd be wary of here. there's a

```javascript
document.body.appendChild(iframe);
```

call in the JS. but as far as I can tell from looking at this, that JS would be executed before we even have a body - we're still in the `<head>` at this point.

Does this not cause any issues?

Even if not - could we not put the `<iframe>` directly into the HTML? Since we're embedding stuff serverside. If you give it an id you should still be able to bind up the event handler in the JS

Reply:
There is a `if (document.body) {` check in the script, and when George and I were pairing on this we did think about that, but we checked and in the context we are using it, there is always body at the point when this gets called. 

We could add the iFrame directly, I believe the point of doing it like this is that you could add the script to any template and it would work, without you having to manually add the iFrame in. I guess if we forget about others using this script then that would make a little more sense, slightly less javascript to run

Reply:
So we don't call `loadFonts()` until the `DOMContentLoaded` event fires in the DOM (see line 44 in the script